src/components/templates/Jazzbar/JazzTab/constants.ts

```diff
@@ -1,13 +1,5 @@
 import { Table } from "types/Table";
 
-const jazzBarTable = (index: number): Table => ({
-  capacity: 6,
-  title: `Table ${index}`,
-  reference: `jazzbar-table-${index}`,
-  rows: 2,
-  columns: 3,
-});
+import { generateTables } from "utils/table";
 
-export const JAZZBAR_TABLES: Table[] = new Array(12)
-  .fill(0)
-  .map((_, index) => jazzBarTable(index + 1));
+export const JAZZBAR_TABLES: Table[] = generateTables({ num: 12, capacity: 6 });
```

Comment:
In the new code we have different logic used for the `reference` key. How is that used by the app, and does it matter? Will the new way work fine?

Reply:
That was the exact reason people got kicked out of tables. Reference is like an id.

![image](https://user-images.githubusercontent.com/63194656/114659713-c2deb900-9cfc-11eb-8cd9-a81733c8acd2.png)


You can find more in src/components/molecules/TablesUserList/TablesUserList.tsx

Reply:
So, tables generated by script had different IDs

Reply:
Huh, interesting!

Reply:
Is this reference used for anything else? Eg. Anything related to twilio/video sessions/etc?

I feel like there possibly was a reason we didn’t use titles for it before (or perhaps we used to and then changed the code so that we no longer did?)

Reply:
@mike-lvov I’ve asked you this a few times in the past, but can you please make sure you include an ‘actual’ description in PRs that talks about what you did/why/etc. Linking to the related issue is insufficient.

It would have saved both of us time here, as it would have set the right context/described things up front.

Reply:
1) Yeah, table.reference is used in getting videoToken 
![image](https://user-images.githubusercontent.com/63194656/114661377-8fe9f480-9cff-11eb-8de6-84e147112bc6.png)

2) I don't think using titles is/was a good idea (the same reason I described above)

Reply:
Dupes are particularly harmful with the table reference thing too, as the way they are used will end up with anyone using the same firebase account, with the same name, ending up in the same video session I believe.

Reply:
Agreed that name is a bad choice to use here. We probably want something like the firebase environment id + venueId + tableId or similar to ensure its unique/isolated/etc.

Reply:
VenueId is indeed used in creation of roomName it's venueId + table reference. We have an issue, which will be implemented during this sprint, which requires adding new functionality to tables. I will have to refactor the table component to be able to add new things and will implement it there

Reply:
Cool cool, sounds good.